Makefile

```diff
@@ -117,9 +117,9 @@ registry-container: generate-manifests-tree
 	$(IMAGE_BUILD_CMD) build --no-cache -f openshift-ci/Dockerfile.registry.upstream.dev -t "$(FULL_REGISTRY_IMAGE)" --build-arg FULL_OPERATOR_IMAGE="$(FULL_OPERATOR_IMAGE)"  .
 
 .PHONY: index-container
-index-container: generate-manifests-index
+index-container: generate-index-database
 	@echo "Building the performance-addon-operator index image"
-	$(IMAGE_BUILD_CMD) build --no-cache -f openshift-ci/Dockerfile.index.upstream.dev -t "$(FULL_INDEX_IMAGE)" .
+	$(IMAGE_BUILD_CMD) build --no-cache -f build/_output/index.Dockerfile -t "$(FULL_INDEX_IMAGE)" build/_output
```

Comment:
this `index.Dockerfile` comes from the `opm` tool, right?

Reply:
yep exactly

Reply:
nope, it not, for the OpenShift CI, we already have the relevant PRs merged.